### DISCUSSION TITLE
import error

### DISCUSSION THREAD

**Discussion by shu-a-be** [lnccbrown/HSSM#279] [Q&A]
Hi
I installed this useful tool using arm64, but I encountered the following issue:
cssm.cpython-311-darwin.so' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64')).
If anyone knows how to resolve this, please let me know.

**Comment by digicosmos86** [comment]
Hi,

Did you try installing `hssm` in a new virtual environment? This error shows that some of the dependencies of `hssm` is still the x86 version instead of arm64. I would create a new virtual environment, and install hssm using `pip install hssm --no-cache-dir` to make sure that you are installing all dependencies afresh.

Let us know if this works.

Thanks!
Paul

**Reply by shu-a-be** [reply]
Hi, Paul.
Thank you for the advice.
I've successfully installed it.
I want to do a lot using this wonderful system.

Thank you so much!
